modules/odc-accessing-perspectives.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// web_console/web-console-overview.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-perspectives_{context}"]
+= Accessing the Perspectives
+
+
+You can access the *Administrator* and *Developer* perspective from the web console as follows:
+
+.Prerequisites
+To access a perspective, ensure that you have logged in to the web console. Your default perspective is determined by your role.
```

Comment:
not sure if we need to get into details of what these role is 

@jerolimov WDYT?

Reply:
Yeah this might be interesting to some users, we've got this question also in the past from associates who uses the web console UI less often. Wdyt:

```suggestion
To access a perspective, ensure that you have logged in to the web console. Your default perspective is automatically determined by the permission of the users. The *Administrator* perspective is selected for users with access to all projects, while the *Developer* perspective is selected for users with limited access to just their own projects.
```

Reply:
Thanks @jerolimov , this looks better IMO

Reply:
Thank you @jerolimov. Additionally I'm removing "just" from from the suggested changes.